static/bidder-info/coinzilla.yaml

```diff
@@ -0,0 +1,10 @@
+maintainer:
+  email: "technical@sevio.com"
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+  site:
+    mediaTypes:
+      - banner
+
```

Comment:
Just to confirm, your bidder doesn't support usersyncing capability?

Reply:
It doesn't support.